mobile/v1/redshift/sql-runner/sql/standard/01-base/01-main/08a-page-view-context.sql

```diff
@@ -0,0 +1,43 @@
+/*
+   Copyright 2021 Snowplow Analytics Ltd. All rights reserved.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+{{if eq .model "web"}}
+
+  DROP TABLE IF EXISTS {{.scratch_schema}}.web_events_addon_pv_context{{.entropy}};
```

Comment:
Nit: I would name this web page context, or just 'page' context.. 

It represents a 'page' (which is a persistent entity),  a 'page view' (which is an occurance in a particular moment in time). ('screen context' vs 'screen view context' would be the analogous case for mobile).

I am probably being v picky here, not massively fussed :)

Reply:
Changed to web_events_addon_page_context